README.rst

```diff
@@ -106,87 +106,6 @@ Therefore, if you want to call ``num2words`` with a fallback, you can do::
 Additionally, some converters and languages support other optional arguments
 that are needed to make the converter useful in practice.
 
```

Comment:
Please make a mention here to check the Wiki for specific language information as FI and JA, so other people making contributions can propose to add more information into the wiki.